.github/workflows/corretto.yml

```diff
@@ -0,0 +1,36 @@
+name: Corretto CI
```

Comment:
can you explain why we add a `corretto.yml` file?

Reply:
I just copied the latest github workflows from Corr

Reply:
We don't have a corretto.yml file in the project template https://github.com/micronaut-projects/micronaut-project-template/tree/master/.github/workflows ?

Are we testing against corretto.yml  in the R2DBC project for any specific reason?

Reply:
No, it was a mistake